DQMOffline/Trigger/python/B2GMonitoring_cff.py

```diff
@@ -1,4 +1,45 @@
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.JetMETMonitor_cfi import hltJetMETmonitoring
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+
+# HLT_AK8PFHT700_TrimR0p1PT0p03Mass50                                                                                                                                         
+AK8PFHT700_HTmonitoring = hltHTmonitoring.clone()
+AK8PFHT700_HTmonitoring.FolderName = cms.string('HLT/B2GMonitor/AK8PFHT700_TrimR0p1PT0p03Mass50')
+AK8PFHT700_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_AK8PFHT700_TrimR0p1PT0p03Mass50_v*")
```

Comment:
@mrosi @rappoccio @alefisico This was my mistake. AK8PFHT700_TrimR0p1PT0p03Mass50 can be removed along with HLT_AK8PFJet360_TrimMass30 , HLT_PFHT650_WideJetMJJ900DEtaJJ1p5  and HLT_PFHT800 these are the old triggers. They need to be replaced with the new list :  'HLT_AK8PFJet360_TrimMass30',   'HLT_AK8PFHT750_TrimMass50', 'HLT_AK8PFJet380_TrimMass30', 'HLT_AK8PFHT800_TrimMass50', 'HLT_AK8PFJet400_TrimMass30','HLT_AK8PFHT850_TrimMass50', 'HLT_AK8PFJet420_TrimMass30','HLT_AK8PFHT900_TrimMass50'